HelpSource/scdoc.css

```diff
@@ -514,12 +524,12 @@ td p {
     overflow: auto;
 /*    background: #eee;*/
     background: #fff;
-    border-right: 1px solid #bbb;
+    border-right: 1px solid var(--color-fg-100);
 }
 
 #sidetoc a:hover {
     text-decoration: none;
-    color: black;
+    color: var(--color-bg);
 }
```

Comment:
I think we can safely remove `#sidetoc` and `#sidetoc a:hover` rules, I can't find any reference to these ids anymore, either in HelpSource or in my compiled helpTargetDir. 
I would delete them all together, but if you want to be more conservative you could just comment them out with a note saying something like "unused?".

The reason I looked into this is because #sidetoc has an hardcoded background color (#fff)

Reply:
If it is not used anymore then we should remove it.